config/yurtctl-servant/setup_edgenode

```diff
@@ -119,11 +120,18 @@ setup_yurthub() {
         log "setting up yurthub on nodes of minikube"
         yurthub_yaml=$(echo "$YURTHUB_TEMPLATE" | 
             sed 's|__pki_path__|/var/lib/minikube/certs|')
+        apiserver_addr=https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT_HTTPS
     else
         log "setting up yurthub on nodes of ack"
         yurthub_yaml=$(echo "$YURTHUB_TEMPLATE" | 
             sed 's|__pki_path__|/etc/kubernetes/pki|')
-    fi 
+        apiserver_addr=`cat $KUBELET_CONF | grep "server:" | awk  '{print $2}'`
```

Comment:
how about get apiserver_addr from local ~/.kube/config, because user has specified it when using yurtctl tool.

Reply:
The server addr of kubelet.conf can ensure that the edge node can access. In order to ensure the accuracy,  the method of getting kubelet.conf is modified from the default /etc/kubernetes/kubelet.conf to get it from the kubelet cmdline.